shared/reducers/chat2.js

```diff
@@ -126,6 +127,19 @@ const metaMapReducer = (metaMap, action) => {
       return ['inboxSyncedClear', 'leftAConversation'].includes(action.payload.reason)
         ? metaMap.clear()
         : metaMap
+    case Chat2Gen.updateConvRetentionPolicy:
+      const update = action.payload.update
+      if (!update.conv) {
```

Comment:
why allow this?

Reply:
I think this would only happen for old subteams that have no conversations set up; in which case nobody using the client would be able to see any retention controls / try to interact with policies (implicit admins can't). Implicit admins might be getting this notification, though, so I figured it'd be best to silently warn

Reply:
Actually reading over that it might be that implicit admins don't get the notice. @mlsteele do they?

Reply:
nevermind I was thinking about this all wrong. the notification handler should check for `update.conv` and request it if it's not present, and I'll change `updateConvRetentionPolicy` to make the inboxUIItem non-optional